src/main/java/LineTokenizer.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class LineTokenizer {
+    Map<String, ArrayList<Integer>> newMapWords;
+    List<Integer> linesWithWord;
+
+    public Map<String, ArrayList<Integer>> mapWords(List<String> lines) {
+        newMapWords = new HashMap<>();
+        lines.stream()
+                .map(line -> line.trim().split("\\s+"))
```

Comment:
You are using the `trim()` method in 4 places, please fix it.

![image](https://user-images.githubusercontent.com/27965136/147774211-d2796e64-d89b-4d11-bd50-c0c99bdba71d.png)

Reply:
now it used in 2 places